std/file.d

```diff
@@ -3,14 +3,13 @@
 /**
 Utilities for manipulating files and scanning directories. Functions
 in this module handle files as a unit, e.g., read or write one _file
-at a time. For opening files and manipulating them via handles refer
-to module $(LINK2 std_stdio.html,$(D std.stdio)).
+at a time.
```

Comment:
What about leaving the original link in the text, in addition to adding it to `See_Also`? It seems important enough to be mentioned directly in the text, not just relegated to the `See_Also` section.